src/sql/parts/availabilityGroup/media/availabilitygroup.css

```diff
@@ -0,0 +1,271 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+availabilitygroupview-component {
+	height: 100%;
+	width: 100%;
+	display: block;
+	font-size: 13px;
+	user-select: text;
+}
+
+.ag-dashboard-top-container {
+	overflow-x: scroll;
+	overflow-y: scroll;
+	padding: 10px;
+	height: 50%;
+}
+
+.ag-tile-container {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: flex-start;
+	min-width: 250px;
+}
+
+.ag-tile {
+	overflow: hidden;
+	cursor: pointer;
+	margin: 10px;
+	max-width: 240px;
+	max-height: 250px;
+	border-style: solid;
+	border-width: 2px;
+	border-color: lightgray;
+	padding: 5px;
+	padding-top: 0px;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+.selected-tile {
```

Comment:
can you scope this class to the AG feature area? either with a more specific name or adding an additional scope selector?